src/vsr.zig

```diff
@@ -1520,12 +1520,22 @@ pub const Checkpoint = struct {
         assert(valid(checkpoint));
 
         if (trigger_for_checkpoint(checkpoint)) |trigger| {
-            return trigger + constants.pipeline_prepare_queue_max;
+            return trigger + (2 * constants.pipeline_prepare_queue_max);
```

Comment:
Now that we can prepare, but not prepare_ok a prepare, let's maybe introduce a `fn send_prepare_ok_after_checkponit`, which we call after checkpoint to send oks for prepares we withheld?

Reply:
Ah, good catch, I thought that we could just rely on the primary's _on_prepare_timeout_ such that it requests the prepare_oks that it needs. It would be much better for replicas proactively send prepare_oks after they are done writing checkpoint to superblock!

Reply:
cecee4531bb3588085d35ecf1caf80d17da0995a